### PR TITLE
Mlw fix release 10.13 10.15

### PR DESCRIPTION
**Pull request by amsosag**
PR to update announcements for mlw 10.13 & 10.15 releases

### CODE REVIEW

File: content/release-10-13-0/announcements-10-13-0.md [hunk 1]
```diff
@@ -171,3 +171,48 @@ The following applies as of Apama 10.11.1:
...
+### Machine learning Workbench
+
+#### Implemented
+
+- Currently both "Machine Learning Manager" & "Machine Learning Admin" user groups have access to execute the python scripts, run python commands using jupyter notebook, train workflows & train neural network models.
+To further enhance the security, the above functionalities will be limited only to "Machine Learning Admin" user group.
+
+- "Machine Learning Admin" user group should be treated as privileged user group.
+
+- This access limitation will be a breaking change for existing "Machine Learning Manager" user group.
```
Comment (BeateRixen) [inline content/release-10-13-0/announcements-10-13-0.md:184]:
```suggestion
- This access limitation will be a breaking change for the existing "Machine Learning Manager" user group.
```

File: content/release-10-13-0/announcements-10-13-0.md [hunk 2]
```diff
@@ -171,3 +171,48 @@ The following applies as of Apama 10.11.1:
...
+
+- "Machine Learning Manager" user group has been restricted from accessing below post apis:
```
Comment (BeateRixen) [inline content/release-10-13-0/announcements-10-13-0.md:186]:
```suggestion
- The "Machine Learning Manager" user group has been restricted from accessing the following post APIs:
```

File: content/release-10-13-0/announcements-10-13-0.md [hunk 3]
```diff
@@ -171,3 +171,48 @@ The following applies as of Apama 10.11.1:
...
+
+    ##### In Training Workflow:
```
Comment (BeateRixen) [inline content/release-10-13-0/announcements-10-13-0.md:188]:
```suggestion
In training workflows:
```

File: content/release-10-13-0/announcements-10-13-0.md [hunk 4]
```diff
@@ -171,3 +171,48 @@ The following applies as of Apama 10.11.1:
...
+    ```
+    {{url}}/service/mlw/projects/{{projectID}}/resources/{{resourcesID}}/workflow
+    ```
+    ##### In Code Execution:
```
Comment (BeateRixen) [inline content/release-10-13-0/announcements-10-13-0.md:192]:
```suggestion
In code execution:
```

File: content/release-10-13-0/announcements-10-13-0.md [hunk 5]
```diff
@@ -171,3 +171,48 @@ The following applies as of Apama 10.11.1:
...
+    ```
+    {{url}}/service/mlw/projects/{{projectID}}/resources/{{resourcesID}}/execute
+    ```
+    ##### In NeuralNetwork:
```
Comment (BeateRixen) [inline content/release-10-13-0/announcements-10-13-0.md:196]:
```suggestion
In neural networks:
```

File: content/release-10-13-0/announcements-10-13-0.md [hunk 6]
```diff
@@ -171,3 +171,48 @@ The following applies as of Apama 10.11.1:
...
+    ```
+    {{url}}/service/mlw/projects/{{projectID}}/resources/{{resourcesID}}/trainNN
+    ```
+    
+- Below apis are only accessible to  "Machine Learning Admin" user group:
```
Comment (BeateRixen) [inline content/release-10-13-0/announcements-10-13-0.md:201]:
```suggestion
- The following APIs are only accessible to  the "Machine Learning Admin" user group:
```

File: content/release-10-13-0/announcements-10-13-0.md [hunk 7]
```diff
@@ -171,3 +171,48 @@ The following applies as of Apama 10.11.1:
...
+
+    ##### GET - jnb-sessions:
```
Comment (BeateRixen) [inline content/release-10-13-0/announcements-10-13-0.md:203]:
```suggestion
GET - jnb-sessions:
```

File: content/release-10-13-0/announcements-10-13-0.md [hunk 8]
```diff
@@ -171,3 +171,48 @@ The following applies as of Apama 10.11.1:
...
+    ```
+    {{url}}/service/mlw/jnb-sessions
+    ```
+    ##### GET - jnb-content:
```
Comment (BeateRixen) [inline content/release-10-13-0/announcements-10-13-0.md:207]:
```suggestion
GET - jnb-content:
```

File: content/release-10-13-0/announcements-10-13-0.md [hunk 9]
```diff
@@ -171,3 +171,48 @@ The following applies as of Apama 10.11.1:
...
+    ```
+    {{url}}/service/mlw/projects/{{projectID}}/resources/{{resourceID}}/jnb-content
+    ```
+    ##### PUT - jnb-content:
```
Comment (BeateRixen) [inline content/release-10-13-0/announcements-10-13-0.md:211]:
```suggestion
PUT - jnb-content:
```

File: content/release-10-13-0/announcements-10-13-0.md [hunk 10]
```diff
@@ -171,3 +171,48 @@ The following applies as of Apama 10.11.1:
...
+    ```
+    {{url}}/service/mlw/projects/{{projectID}}/resources/{{resourceID}}/jnb-content
+    ```
+
+- "Machine Learning Manager" user group has been restricted from executing jupyter notebooks & accessing assets section.
```
Comment (BeateRixen) [inline content/release-10-13-0/announcements-10-13-0.md:216]:
```suggestion
- The "Machine Learning Manager" user group has been restricted from executing Jupyter notebooks and accessing the assets section.
```

File: content/release-10-13-0/announcements-10-13-0.md [hunk 11]
```diff
@@ -171,3 +171,48 @@ The following applies as of Apama 10.11.1:
...
+
+- "Machine Learning User" user group has been restricted from accessing assets section.
```
Comment (BeateRixen) [inline content/release-10-13-0/announcements-10-13-0.md:218]:
```suggestion
- The "Machine Learning User" user group has been restricted from accessing the assets section.
```

**Comment by BeateRixen** [comment]
@amsosag I have done a review on the text for 10.13. Please adapt the 10.15 text accordingly.

File: content/release-10-13-0/announcements-10-13-0.md (hunk 1 above)
Comment (BeateRixen) [inline content/release-10-13-0/announcements-10-13-0.md:184]: I suggest to provide some recommendation to the users what to do

**Comment by BeateRixen** [comment]
@amsosag I just realized that this is supposed to go into the 10.13 announcements. This is no longer possible, we cannot announce breaking changes for releases that already went live.

File: content/release-10-13-0/announcements-10-13-0.md (hunk 1 above)
Comment (amsosag) [reply content/release-10-13-0/announcements-10-13-0.md]: done

**Comment by amsosag** [comment]
> @amsosag I just realized that this is supposed to go into the 10.13 announcements. This is no longer possible, we cannot announce breaking changes for releases that already went live.

For now I have removed the changes from 10.13 announcements until further clarification from team. This PR now only has changes in 10.15 important announcement section.

**Comment by amsosag** [comment]
> @amsosag I have done a review on the text for 10.13. Please adapt the 10.15 text accordingly.

I have implemented the suggested changes in 10.15 important announcement section.

File: content/release-10-15-0/announcements-10-15-0.md [hunk 12]
```diff
@@ -265,3 +265,54 @@ This is available from the following web page: https://documentation.softwareag.
...
+    {{url}}/service/mlw/projects/{{projectID}}/resources/{{resourcesID}}/trainNN
+    ```
+
+- The "Machine Learning Manager" user group has been restricted from executing Jupyter notebooks and accessing the assets section as well.
+
+- The "Machine Learning User" user group has been restricted from accessing the assets section.
+
+- This access limitation will be a breaking change for the existing "Machine Learning Manager" user group.
+
+- Those users who are currently part of "Machine Learning Manager" group and needs code execution access will need to be a part of higher privileged group, i.e. "Machine Learning Admin".
```
Comment (BeateRixen) [inline content/release-10-15-0/announcements-10-15-0.md:318]:
```suggestion
- Those users who are currently in the "Machine Learning Manager" group and need code execution access must be accordingly added to a higher privileged group, for example, "Machine Learning Admin".
```

**Review by BeateRixen — APPROVED** [review]

File: content/release-10-15-0/announcements-10-15-0.md [hunk 13]
```diff
@@ -265,3 +265,54 @@ This is available from the following web page: https://documentation.softwareag.
 Further to the CEL (Esper) deprecation notice in [release 10.5](/release-10-5-0/migration-10-5-0/) and the subsequent announcement of end of support in [release 10.7](/release-10-7-0/announcements-10-7-0/) it has been possible to continue using Esper in an unsupported mode. With effect from release 10.15 the CEL (Esper) functionality has been removed completely and cannot be used anymore.
 
 If you need assistance to migrate your streaming analytics logic from Esper to Apama, please contact [Software AG Global Support](/about/contacting-support/).
+
+### Machine learning Workbench
```
Comment (BeateRixen) [inline content/release-10-15-0/announcements-10-15-0.md:269]:
```suggestion
### Machine Learning Workbench
```

**Review by VRAN016 — APPROVED** [review]

**Pull request merged**